### PR TITLE
Adjusting README.md

### PR DESCRIPTION
**Pull request by Koe-Nocturne**
Including a link of the pyenv installation instructions. When setting up I found that pip was not showing up as installed so had to perform step 3 in the link that I added to the README.md. First test commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ memcached -u memcached -d -m 30 -l 127.0.0.1 -p 11211
 Make sure you are in the project directory.
 
 
-Install the latest version of Python with `pyenv`
+Install the latest version of Python with `pyenv` if you have not installed `pyenv` before make sure to follow the instructions [here](https://github.com/pyenv/pyenv#installation).
```
Comment (kaedub) [inline README.md]: You can just add `brew install pyenv` to line 20-23 since it's installed the same way sqlite3 and memcached are.

**Comment by kaedub** [comment]
@Koe-Nocturne thanks for being the setup guinea pig!🥇

File: README.md (hunk 1 above)
Comment (Koe-Nocturne) [reply README.md]: I installed pyenv but I needed step 3 in the link. Should I just include that step in the install section?

Comment (kaedub) [reply README.md]: Oh I see, good find! If that was required for your setup, then definitely add that as well.

**Pull request merged**